react/StoreContextProvider.js

```diff
@@ -13,7 +13,7 @@ import { PixelProvider } from './PixelContext'
 const APP_LOCATOR = 'vtex.store'
 const CONTENT_TYPE = 'text/html;charset=utf-8'
 const META_ROBOTS = 'index, follow'
-const MOBILE_SCALE_OFF = 'width=device-width, initial-scale=1, maximum-scale=1, user-scalable=0'
+const MOBILE_SCALE_OFF = 'width=device-width, initial-scale=1, maximum-scale=5'
```

Comment:
1 - o nome da variavel poderia ser `INITIAL_SCALE` ou `MOBILE_SCALING` agora, já que nao ta desativando o redimensionamento

2 - nem precisaria dar uma maximum-scale
```suggestion
const MOBILE_SCALE_OFF = 'width=device-width, initial-scale=1'
```